### PR TITLE
[READY] linuxraw: Create Wireless Controller.cfg

### PR DESCRIPTION
**Pull request by davidhedlund**

### CODE REVIEW

File: linuxraw/Wireless Controller.cfg [hunk 1]
```diff
@@ -0,0 +1,61 @@
+input_driver = "linuxraw"
+# The files "Sony Interactive Entertainment Wireless Controller.cfg" and "Wireless Controller.cfg" should have identical content except for the input_device variable. This difference allows thelinuxraw driver to recognize the controller on various Linux kernel versions.
+input_device = "Wireless Controller"
+input_device_display_name = "Sony DualShock 4 v2 Controller"
```
Comment (RobLoach) [inline linuxraw/Wireless Controller.cfg:4]: Are you able to get the input_vendor_id and input_product_id?

Comment (davidhedlund) [reply linuxraw/Wireless Controller.cfg:4]: The linuxraw driver does not utilize the Vendor ID and Product ID. This is thoroughly explained in the documentation available at https://docs.libretro.com/guides/controller-autoconfiguration/.

**Comment by davidhedlund** [comment]
Merged to https://github.com/libretro/retroarch-joypad-autoconfig/pull/1172/files

**Comment by davidhedlund** [comment]
I've merged **all** my PRs to simply the reviews for you.

**Pull request closed without merging**